ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/prefix-lists.j2

```diff
@@ -2,8 +2,8 @@
 {% for prefix_list in prefix_lists | arista.avd.natural_sort('name') %}
 !
 ip prefix-list {{ prefix_list.name }}
-{%     for sequence in prefix_list.sequence_numbers | arista.avd.natural_sort('sequence') %}
-{%         if sequence.action is arista.avd.defined %}
+{%     for sequence in prefix_list.sequence_numbers | arista.avd.default([]) | arista.avd.natural_sort('sequence') %}
```

Comment:
Natural sort already defaults to an empty list, so you can skip it where you have sorting.

Reply:
fixed!